index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+        rel="stylesheet">
+    <link rel="stylesheet" href="index.css">
+    <title>mobile skeleton</title>
+</head>
+<body>
+    <header>
+       <!-- logo left -->
+        <div class="logo">
+            <h3>My Logo</h3>
+        </div>
+        <!-- strart right nav-bar -->
+        <div class="nav">
+            <i id="menu" class="material-icons">menu</i>
+        </div>
```

Comment:
- [ ] Kindly replace this `<div class='nav'>` tag with a `<nav>` tag to make proper use of semantic HTML.